app/tests/reader_studies_tests/test_api.py

```diff
@@ -1692,3 +1701,111 @@ def test_display_set_delete(client):
     )
 
     assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_total_edit_duration(client):
+    rs = ReaderStudyFactory(allow_answer_modification=True)
+    ds = DisplaySetFactory(reader_study=rs)
+    q = QuestionFactory(
+        reader_study=rs, answer_type=AnswerType.SINGLE_LINE_TEXT
+    )
+    u = UserFactory()
+
+    rs.add_reader(u)
+
+    response = get_view_for_user(
+        viewname="api:reader-studies-answer-list",
+        user=u,
+        client=client,
+        method=client.post,
+        content_type="application/json",
+        data={
+            "question": q.api_url,
+            "display_set": ds.api_url,
+            "answer": "foo",
+        },
+    )
+
+    assert response.status_code == 201
+    assert response.json()["last_edit_duration"] is None
+    assert response.json()["total_edit_duration"] is None
+
+    pk = response.json()["pk"]
+
+    response = get_view_for_user(
+        viewname="api:reader-studies-answer-detail",
+        user=u,
+        client=client,
+        method=client.patch,
+        reverse_kwargs={"pk": pk},
+        content_type="application/json",
+        data={"answer": "bar", "last_edit_duration": "00:30"},
+    )
+    assert response.status_code == 200
+    assert response.json()["last_edit_duration"] == "00:00:30"
```

Comment:
Is this correct? Shouldn't we ignore the value on patch if the total is `None`?

Reply:
I thought it would make more sense to handle the value that has been patched, but just not update the total, since that is a caclulated value. I'd be happy to change it though.

Reply:
I see, yes I think this makes sense.